playwright-e2e/page-objects/local-authority/placement/laPlacementAndCourtsSection.page.ts

```diff
@@ -0,0 +1,13 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../../pages/basePage.page';
+
+export class LAPlacementOrdersInPlacePage extends BasePage {
+  readonly placementOrdersInPlaceHeading: Locator;
+
+  constructor(page: Page) {
+    super(page);
+
+    this.placementOrdersInPlaceHeading = page.getByRole('heading', { name: 'Orders already in place' });
```

Comment:
is anywhere the heading is variable? its declared but not used.

Reply:
This is to be utilised in future as a check when in the LA journey